resources/views/metadata-tags.blade.php

```diff
@@ -1,12 +1,15 @@
 @props([
     'title',
     'description',
+    'metaTitle' => null,
     'image' => null,
 ])
 
+<title>{{ $title }}</title>
```

Comment:
why was this needed? Title has no issues for me on that blog 🤔 
![image](https://user-images.githubusercontent.com/35610748/130631336-75283e18-44e9-411e-b1b0-c2b181e5e4f3.png)

Reply:
@ItsANameToo is that local?, the production site has the issue
![image](https://user-images.githubusercontent.com/17262776/130633725-2fe69d28-4417-47a7-9d0f-08e00e683d5b.png)

`view-source:https://marketsquare.io/projects/marketsquare/blog/i-spent-6-months-inside-the-metaverse-and-heres-why-it-needs-crypto`

Reply:
and I have it locally (before the patch)

Reply:
it's on production, but in devtools. looks like `view-source:` encodes it :sus:

Reply:
@ItsANameToo interesting... seems that devtools encodes text inside nodes but not regular attributes like the `og:title` I suppose is because is what you usually see

Ill check if the title alone affect the sharing maybe you are right and is not necessary